src/main/java/com/faforever/api/voting/VotingService.java

```diff
@@ -100,7 +101,9 @@ public void saveVote(Vote vote, Player player) {
       errors.add(new Error(ErrorCode.VOTE_ALREADY_ENDED, subject.getEndOfVoteTime()));
     }
 
-    if (gamesPlayed < subject.getMinGamesToVote()) {
+    boolean accountQualifiedBySteamAndAge = player.getSteamId() != null && OffsetDateTime.now().minusYears(ACCOUNT_AGE_YEARS_VOTE_QUALIFIED).isAfter(player.getCreateTime());
+
+    if (!accountQualifiedBySteamAndAge && gamesPlayed < subject.getMinGamesToVote()) {
       errors.add(new Error(ErrorCode.NOT_ENOUGH_GAMES, gamesPlayed, subject.getMinGamesToVote()));
```

Comment:
Should we change error message too?

Reply:
No, **if** the error occurs its still correct. It just occurs in less cases.

Reply:
wah, you are right